data/daos/implementations/ArrayListStudentDao.java

```diff
@@ -0,0 +1,53 @@
+package data.daos.implementations;
+
+import data.daos.StudentDao;
+import data.daos.exceptions.NoSuchStudentExist;
+import logic.models.Student;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static java.util.Arrays.stream;
+
+public class ArrayListStudentDao implements StudentDao {
+    private final ArrayList<Student> students;
+
+    public ArrayListStudentDao() {
+        students = new ArrayList<>();
+    }
+
+    @Override
+    public void add(Student student) {
+        students.add(student);
+    }
+
+    @Override
+    public void remove(String idToRemove) {
+        Student studentToRemove=students.stream()
+                .filter(s -> s.getId().equals(idToRemove))
+                .findFirst().
+                orElseThrow(() -> new NoSuchStudentExist(idToRemove));
+       students.remove(studentToRemove);
+<<<<<<< HEAD
```

Comment:
I think you forgot to remove something leftover from a merge conflict.

Reply:
Done Raimund.